docs/proposals/config_data.md

```diff
@@ -0,0 +1,298 @@
+# Generic Configuration Object
+
+## Abstract
+
+This proposal proposes a generic configuration object named ``ConfigData`` stored inside ``etcd`` that
```

Comment:
@tmrts as @bgrant0607 mentioned earlier, single backticks are sufficient for monospaced font -- our convention is to just use the single backticks.  Plz change.